src/views/Detail/Detail.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { selectDetail } from '../../services/bobs-burgers';
+import DetailCard from '../../components/DetailCard/DetailCard';
+import { useHistory } from 'react-router-dom';
+
+export default function Detail() {
+  const [loading, setLoading] = useState(true);
+  const [selected, setSelected] = useState([]);
```

Comment:
We should initialize the state to what we expect to become. In this case an object